### ISSUE TITLE
Dragging Windows with two monitor not working well with a vertical monitor

### ISSUE THREAD

**Issue by vincenzopalazzo** [vincenzopalazzo/material-ui-swing#144]
This issue is related to another issue (#128)

The problem that I discovered maybe is inside the following method

```java
    public MaterialRootPaneUI() {
        super();
        int devideWithd = 0;
        int deviceHeight = 0;
        GraphicsDevice graphicDevices[] = GraphicsEnvironment.getLocalGraphicsEnvironment().getScreenDevices();
        for (GraphicsDevice graphicsDevice : graphicDevices) {
            //FIXME: What appened when the one monitor is vertical and not horizzontal?
            devideWithd += graphicsDevice.getDisplayMode().getWidth();
            deviceHeight += graphicsDevice.getDisplayMode().getHeight();
        }
        this.dimensionDevices = new Dimension(devideWithd, deviceHeight);
        this.parentBounds = this.dimensionDevices;
    }
```

## Idea

When the monitor is vertical maybe is possible see the width and height in the opposite meaning?

Somethings like the code above?

```java
Dimension scrnsize = toolkit.getScreenSize();
    if(scrnsize.getWidth() > scrnsize.getHeight()){
         // Do normal x-axis mouse
    }else{
        //  Do inverted y-axis mouse
    }  
```

Othere resources [here](https://jvm-gaming.org/t/detecting-screen-orientation-and-angle/39477/16)

**Comment by zanderson9** [comment]
Hi Vincent,

I have additional info on this issue: I checked with the JMARS team who reported this issue. They verified to me that their monitors are **NOT** in **Vertical** position when they encounter this issue. Their monitor are just side-by-side, in Horizontal orientation. But one display can be larger (taller/wider) than another. For example, a developer may have a built-in laptop display and an external Dell or HP display. Also one of the displays (or both?) are 4K.

**Comment by vincenzopalazzo** [comment]
I'm closing this issue because the result from my test are very crazy. At the moment I don't know if the L&F is the problem or is the system call `JDialog.setDefaultLookAndFeelDecorated(true);` or maybe a both.

**Comment by kenrios** [comment]
Hello Vincent,
      I have some results from testing the demo jar file. 
My display arrangement is that my 4k monitor is my secondary (not vertical) while my laptop display is my primary. I am running Windows 10. The image below shows the difference in height.
![image](https://user-images.githubusercontent.com/9437346/89228362-51a70d80-d594-11ea-96e2-9acaa03b2322.png)

I started the jar file up and followed these steps:
   1. Clicked the "ADD Image" button and attempted to drag the dialog from primary monitor to secondary 4k
         Result: I was unable to move the dialog to the secondary monitor
   2. I opened the info dev dialog from the INFO menu and attempted to drag the dialog from primary to secondary 4k
         Result: I was unable to drag the dialog to the secondary monitor
   3. I changed the style to metal L&F using the style menu
   4. I clicked add image and attempted to drag the file chooser dialog from primary to secondary 4k monitor.
        Result: Success! I was able to move the dialog onto the secondary monitor and move it around
   5. I opened up the Info dev dialog from the INFO menu and attempted to drag from primary to secondary 4k monitor
        Result: Failure. Even with Metal L&F, I was unable to move this dialog to the secondary display. 


I hope this helps!

**Comment by vincenzopalazzo** [comment]
Hi @kenrios,

Yes, your result was amazing, is totally different to see this bug with a different type of monitor. I discussed your issue [here](https://github.com/vincenzopalazzo/material-ui-swing/issues/128). It is a very long conversation. In sum, I fixed this problem but maybe with some JDK and 4k display this issue persists. In addition, I'm able to reproduce another issue with a normal full HD display, so when I drag the windows in the second monitor, the windows disappear but remain under my control. This means that I'm able to return the windows inside the main display.

The issue that I have on my monitor is present also with Metal L&F, but maybe your issue is resolvable is I will try to find the error inside the value the java return from displays. The code inside the dragging is not very clean, or not clean like the code inside the library because is "copy and past" from JDK 8

In conclusion, I can say that at the moment I don't have an idea how to fix this problem bug you can disable the undecorate propriety inside the L&F configuration. With this change, the windows are managed from OS and this resolves all problems that we have. Unfortunately, you will lose the Material Style inside the Dialog :/

**Comment by vincenzopalazzo** [comment]
Solution https://github.com/vincenzopalazzo/material-ui-swing/commit/1d8248d150e7fcbe4fe85f8c764539e67c202603 tested on another swing app with a Swing app coding style. 

This solution, look good at the moment I will make more test before try to join to master
